python/cudf/cudf/api/types.py

```diff
@@ -161,9 +161,9 @@ def is_list_like(obj):
     bool
         Return True if given object is list-like.
     """
-    return isinstance(obj, (Sequence, np.ndarray)) and not isinstance(
-        obj, (str, bytes)
-    )
+    return isinstance(
+        obj, (Sequence, np.ndarray, cp.ndarray)
+    ) and not isinstance(obj, (str, bytes))
```

Comment:
Instead of adding `cp.ndarray` to `is_list_like`, it would be more appropriate to use `is_column_like` where needed. `is_list_like` is being used across code-base to check for list-like objects stored in host memory.